src/fast-csv-wrapper.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash/fp'
+import { format as formatCSV } from '@fast-csv/format'
+import { keysToObject } from './futil'
+import { isIterable, isAsyncIterable } from './utils'
+
+export let format = ({ transformHeaders = x => x, transformedHeaders = null, onWrite = _.noop, includeEndRowDelimiter = true, ...props }) => {
+  let csv = formatCSV({...props, headers: true, includeEndRowDelimiter})
```

Comment:
nit:  a comment will be helpful:`fast-csv will auto-discover headers when the headers option is set to true.`